specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -3223,7 +3239,7 @@
             "Canceled"
           ],
           "x-ms-enum": {
-            "name": "ProvisioningState",
+            "name": "ComputeProvisioningState",
```

Comment:
>ComputeProvisioningState [](http://example.com/codeflow?start=21&length=24)

change this back to provisioningState

Reply:
Same comment on this. This is changing the type name of this enumeration not the property name.
And in the meantime, why we are changing one enumeration into two names (`Workspace` and `Compute`)?  Is it necessary? I personally like that we have both `WorkspaceProvisioningState`, or better `MachineLearningProvisioningState`

Reply:
They are now two different cases as we added the SoftDelete State to the Workspace one, but Compute state don't have this change supported, so I think it's better to name them differently now.

Reply:
Synced up with Xi, if we keep them with the same name there will be a Lint Diff error, we can name them back as the same if you guys think we can ignore that error.

Reply:
@rkmanda do you think we should rename this back to provisioningState even if it will cause Lint Diff error?

Reply:
@ZhidaLiu @ArcturusZhang Looks like ArcturusZhang is right that this should not impact ARM, I'll sign off for ARM, and ArcturusZhang can decide if it's good with the renaming.